pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_command-line-interface.adoc

```diff
@@ -0,0 +1,52 @@
+[id="command-line-interface"]
+
+= Using the Command Line Interface
+
+Some operations available in the Pantheon user interface can also be performed using commands in the command line interface.
+
+.Using the `Pantheon uploader` via the command line
+
+. Install the uploader script via `setup.sh`:
++
+----
+curl -S https://raw.githubusercontent.com/redhataccess/pantheon/master/uploader/setup.sh | sh -
+----
++
+. Type `pantheon` in the command line to confirm it was properly installed.
+
+. Navigate to a directory containing test modules.
++
+[options="nowrap", subs="normal"]
+----
+cd _PATH_TO_YOUR_FOLDER_WITH_MODULES_
+----
+. Create the configuration file and edit as needed.
++
+----
+# pantheon --sample > pantheon2.yml
+# vim pantheon2.yml  (or open in your preferred editor)
+----
+. Change the default values for `server` and `repository` as follows:
++
+[options="nowrap",subs="normal"]
+----
+server: link:http://pantheon2.int.us-west.aws.prod.paas.redhat.com[]
+
+repository: _YOUR_KERBEROS_ID_
+----
+. Add the desired file patterns for modules and resources in the file;
+all file patterns are relative to the configuration file location.
++
+* Modules are pre-viewable files, while resources are files which are necessary to build individual modules.
++
+* Any file matching a pattern in the modules section will show up in the search results.
++
+. Run the `Pantheon uploader` command:
++
+[options="nowrap", subs="normal"]
+----
+pantheon push -u _YOUR_USER_ -p _YOUR_PASSWORD_
+----
+. Locate and preview your modules at:
++
+http://pantheon2.int.us-west.aws.prod.paas.redhat.com/pantheon/
```

Comment:
Let's not expose internal URLs - maybe change this to saying that modules can be located and viewed inside Pantheon V2 itself.